osmdroid-android/src/main/java/org/osmdroid/tileprovider/MapTileProviderBasic.java

```diff
@@ -203,15 +203,18 @@ protected boolean isDowngradedMode(final long pMapTileIndex) {
 	 * @since 6.0.3
 	 * cf. https://github.com/osmdroid/osmdroid/issues/1172
 	 */
-	public static MapTileFileStorageProviderBase getMapTileFileStorageProviderBase(
+	protected MapTileFileStorageProviderBase getMapTileFileStorageProviderBase(
```

Comment:
We can't just do that, for legacy reason.
Please create a new `protected` method if needed but leave the `public static` one.

Reply:
As I saw the original use of this method, the only reason it was static because we used it in `SampleLieFi` and `GeoPackageProvider` for creating `MapTileFileStorageProviderBase ` instances, which I think is unnecessary and blocks the overridability of the method.

You can check it out if you search for the usages of the original `getMapTileFileStorageProviderBase()` method